packages/react-ui-kit/src/Form/Select.tsx

```diff
@@ -65,30 +68,36 @@ export const selectStyle: <T>(theme: Theme, props, error?: boolean) => CSSObject
     textShadow: '0 0 0 #000',
   },
   '&:disabled': {
-    color: COLOR_V2.GRAY,
+    color: theme.select.disabledColor,
   },
   appearance: 'none',
-  background: disabled
-    ? `${theme.Input.backgroundColorDisabled} center right 16px no-repeat url("${inlineSVG(ArrowDown(theme))}")`
-    : `${theme.Input.backgroundColor} center right 16px no-repeat url("${inlineSVG(ArrowDown(theme))}")`,
-  boxShadow: markInvalid ? `0 0 0 1px ${COLOR_V2.RED}` : `0 0 0 1px ${COLOR_V2.GRAY_40}`,
+  boxShadow: markInvalid ? `0 0 0 1px ${COLOR_V2.RED}` : `0 0 0 1px ${theme.select.borderColor}`,
```

Comment:
maybe the invalid color needs to be a Theme variable as well (as it might be different in dark/light mode). Doesn't it?

Reply:
I'll use standard red for now (@red-500)